### ISSUE TITLE
SendGrid Web API port change

### ISSUE THREAD

**Issue by davydav** [eddiezane/sendgrid-actionmailer#82]
![Capture d’écran 2020-07-14 à 16 03 28](https://user-images.githubusercontent.com/272360/87520745-a9ccad00-c683-11ea-97a5-75ba95434b46.png)

Is there anything to do about it in your gem about it ? Hopefully it would only require upgrading the `sendgrid-ruby` gem, but I don't think it's been updated to handle this issue.

**Comment by tyrauber** [comment]
This is really odd. This gem is a wrapper to the sendgrid-ruby gem and only accesses the sengrid api through the sendgrid-ruby gem.  The master uses sendgrid-ruby ~6.0 (which is a little old), but I would be really surprised if it was not using SSL. I highly doubt sendgrid would publish a gem that accesses their API without SSL.  As an additional data point, I've been personally using this gem in production and I have not received this notice.

I've created a branch fix/sendgrid-ruby-6.3 which bumps sendgrid-ruby to master. You are welcome to try that out and see if it resolves the issue, but I suspect something else is going on here.   You might want to reach out to SendGrid Support for clarification?  They should be able to provide specific endpoints. 

Please let us know what you find out.

**Comment by siva-wal** [comment]
We also got this email from SendGrid. I am actually using actionmailer with SMTP configuration as following.

```
  ActionMailer::Base.smtp_settings = {
    :address        => 'smtp.sendgrid.net',
    :port           => '587',
    :authentication => :plain,
    :user_name      => Rails.application.credentials.sendgrid[:username],
    :password       => Rails.application.credentials.sendgrid[:password],
    :enable_starttls_auto => true
  }
```
I don't know why I got this. I reached support but I didn't get any answer from their side.

**Comment by tyrauber** [comment]
Hi @siva-wal, are you using this gem as well, or just using SendGrid raw over SMTP?

**Comment by siva-wal** [comment]
Hi @tyrauber I am not using this gem. But planning to use to avoid this issue. But what I didn't under is, since I am sending mail via SMTP but warning related to port 80. Then why I got this email from SendGrid is my confusion. I wrote that same to their support. Waiting for their reply.

**Comment by eddiezane** [comment]
@thinkingserious do you know if these went out to some folks by mistake?

**Comment by thinkingserious** [comment]
Hello everyone,

My apologies for any confusion. 

tldr; you probably don't need to be concerned.

Please [see this thread](https://github.com/sendgrid/sendgrid-python/issues/928#issuecomment-671520855) for further discussion.

With best regards,

Elmer

Thanks for the heads up @eddiezane!!

**Comment by tyrauber** [comment]
Closing due to inactivity. Please reopen if the problem has persisted.
